.github/workflows/integration_test.yaml

```diff
@@ -5,18 +5,12 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@add_support_for_canonical_k8s
     secrets: inherit
     with:
-      load-test-enabled: false
-      load-test-run-args: "-e LOAD_TEST_HOST=localhost"
-      zap-before-command: "curl -H \"Host: indico.local\" http://localhost/bootstrap --data-raw 'csrf_token=00000000-0000-0000-0000-000000000000&first_name=admin&last_name=admin&email=admin%40admin.com&username=admin&password=lunarlobster&confirm_password=lunarlobster&affiliation=Canonical'"
-      zap-enabled: true
-      zap-cmd-options: '-T 60 -z "-addoninstall jython" --hook "/zap/wrk/tests/zap/hook.py"'
-      zap-target: localhost
-      zap-target-port: 80
-      zap-rules-file-name: "zap_rules.tsv"
-      trivy-fs-enabled: true
-      trivy-image-config: "trivy.yaml"
+      provider: k8s
+      use-canonical-k8s: true
       self-hosted-runner: true
-      self-hosted-runner-label: "edge"
+      self-hosted-runner-label: "large"
+      load-test-enabled: false
```

Comment:
I think it is already by default set to false :D https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml#L60

Reply:
Right, I'm removing it. Thanks!